components/automate-dex/web/templates/login.html

```diff
@@ -2,36 +2,67 @@
 <!-- rendered using https://golang.org/pkg/html/template/-->
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-    <title>{{ issuer }}</title>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link href="{{ url .ReqPath "static/main.css" }}" rel="stylesheet">
-    <link href="{{ url .ReqPath "theme/styles.css" }}" rel="stylesheet">
-  </head>
-  <body>
-    <main class="theme-panel">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+  <title>{{ issuer }}</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href='{{ url .ReqPath "static/main.css" }}' rel="stylesheet">
+  <link href='{{ url .ReqPath "theme/styles.css" }}' rel="stylesheet">
+</head>
+
+<body>
+  {{ if eq ("showBanner" | extra) "true" }}
+  <div class="warning-banner" style='background-color: #{{ "bannerBackgroundColor" | extra }}'>
+    <p class="warning-text" style='color: #{{ "bannerTextColor" | extra }}'>
+      {{ "bannerMessage" | extra }}
+    </p>
+  </div>
+  {{ end }}
+  <div class="theme-panel" role="main">
+    <div class="signin-login-container">
       <div class="logo">
-        <img class="theme-navbar__logo" alt="Chef Automate" src="{{url .ReqPath "static/img/automate-blue-d9789f4b.svg" }}">
+        <img class="theme-navbar__logo" alt="Chef Automate"
```

Comment:
You'll see a few changes like this as well - there is no change in the outcome.  Semgrep was complaining we were not interpolating the template variables properly, so we've updated while we're here.